docs/terms/data_assistant.md

```diff
@@ -0,0 +1,81 @@
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='inactive'/>
+
+## Overview
+
+### Definition
+
+A Data Assistant is a utility that asks questions about your data, gathering information to describe what is observed, and then presents <TechnicalTag tag="metric" text="Metrics" /> and proposes <TechnicalTag tag="expectation" text="Expectations" /> based on the answers.
+
+### Features and promises
+
+Data Assistants allow you to introspect multiple <TechnicalTag tag="batch" text="Batches" /> and create an <TechnicalTag tag="expectation_suite" text="Expectation Suite" /> from the aggregated Metrics of those Batches.  They provide convenient, visual representations of the generated Expectations to assist with identifying outliers in the corresponding parameters.  They are convenient to access from your <TechnicalTag tag="data_context" text="Data Context" />, and provide an excellent starting point for building Expectations or performing initial data exploration.
+
+### Relationships to other objects
+
+A Data Assistant implements a pre-configured <TechnicalTag tag="profiler" text="Rule Based Profiler" /> in order to gather Metrics and propose an Expectation Suite based on the introspection of the Batch or Batches contained in a provided <TechnicalTag tag="batch_request" text="Batch Request" />.
+
+## Use cases
+
+<CreateHeader/>
+
+Data Assistants are an ideal starting point for creating your Expectations.  If you are working with data that you are not familiar with, a Data Assistant can give you an overview by introspecting it and generating a series of relevant Expectations using estimated parameters for you to review. If you use the `"flag_outliers"` value for the `estimation` parameter your generated Expectations will have parameters that disregard values that the Data Assistant identifies as outliers. Using the Data Assistant's `plot_metrics()` method will then give you a graphical representation of the generated Expectations.  This will further assist you in spotting outliers in your data when reviewing the Data Assistant's results.
+
+Even when working with data that you are familiar with and know is good, a Data Assistant can use the `"exact"` value for the `estimation` parameter to provide comprehensive Expectations that exactly reflect the values found in the provided data.
+
+## Features
+
+### Easy profiling
+
+Data Assistants can function as pre-configured Profilers.  Since you can access them directly from your Data Context, they are easily accessible.  This ensures that they will always provide a quick, simple entry point to creating Expectations and <TechnicalTag tag="profiling" text="Profiling" /> your data.  However, the rules implemented by a Data Assistant are fully exposed in the parameters for its `run(...)` method.  This means that while you can use a Data Assistant easily out of the box, you can also customize it behavior to take advantage of the domain knowledge possessed by subject-matter experts.
```

Comment:
@Rachel-Reverie In fact, a Data Assistant is exactly a pre-configured Rule-Based Profiler.  Specifically, the Data Assistant is put together by writing rules that together fit the theme of the given Data Assistant goal at hand.

Reply:
Yes, but all previous versions of profilers have, to the best of my knowledge, been exclusively oriented toward creating an Expectation Suite.  Since we're differentiating from the past behavior (Data Assistants can be used to generate Metrics and plots without creating an Expectation Suite) it seemed it would be more intuitive for our users to say that they "can" be used as a pre-configured Profiler (according to the previous understanding of what a Profiler does and produces).  But they also have additional functionality.

Perhaps it would be better to say: "Data Assistants are preconfigured Profilers with extended versatility.  The rules..." ?